Update CAM Google Cloud doc for instance protection, register and self-register (#2879)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

Instances imported from AWS will inherit DisableApiTermination flag into **Manual Terminate Protection** flag. Flag status is provided when provider is syncronized, so changes are not reflected in CAM until **Sync** operation has been done.

> #### Instance protection on Google Cloud Platform instances

Reviewer: Please, use level 3 title and update ToC and metadata (date and add you as an author, at least):
```suggestion
### Instance protection on Google Cloud Platform instances
```

> ##### Deployed Instances

Reviewer: ```suggestion
#### Deployed Instances
```

> Instances deployed in Google Cloud Platform will sync CAM **Manual Terminate Protection** with DeletionProtection flag in GCP instances. Changes done from CAM will be reflected in GCP Instance Properties. Changes done from GCP Console, won't be reflected in CAM.

Reviewer: ```suggestion
Instances deployed in Google Cloud Platform will sync Cloud Application Manager's **Manual Terminate Protection** flag with the _DeletionProtection_ flag in GCP instances. Changes done to this flag through Cloud Application Manager interface will be reflected in GCP Instance Properties. Changes done from GCP Console to the _DeletionProtection_ flag will not be reflected in Cloud Application Manager instances.
```

**Note:** *compute.instances.create* permission or one of the roles *compute.admin* or *compute.instanceAdmin.v1* must be enabled in order to modify Instance Flag

> ##### Unregistered Instances

Reviewer: ```suggestion
#### Unregistered Instances
```

> Instances imported from GCP will inherit DeletionProtection flag into **Manual Terminate Protection** flag. Flag status is provided when provider is syncronized, so changes are not reflected in CAM until **Sync** operation has been done.

Reviewer: Hey Efren, does registered instances also sync the value of the flag if you sync the provider? I would clarify this by mentioning if the sync updates only unregistered instances or both registered and unregistered.
```suggestion
Instances imported from GCP will inherit DeletionProtection flag into **Manual Terminate Protection** flag in Cloud Application Manager. Flag status is obtained when the provider is syncronized, so the changes performed in GCP Console are not reflected in Cloud Application Manager until **Sync** operation has been done.
```
